Shelfie/Shelfie/UI/BookDetail/BTAddBookViewController.m

```diff
@@ -51,6 +51,7 @@ - (void)makeBook:(NSDictionary *)book {
     NSDictionary *bookDictionary = book[@"items"][0][@"volumeInfo"];
     NSError *error;
     self.book = [[GOBook alloc] initWithDictionary:bookDictionary error:&error];
+    // set up all of the views etc in the rest of this function
```

Comment:
Add TODO tag for unfinished feature, like:
'//TODO: set up all of the views etc in the rest of this function'